[SDTEST-426] docs for agentless logs submission in Ruby (#28773)
DataDog/documentation: content/en/tests/correlate_logs_and_tests/_index.md

| `DD_AGENTLESS_LOG_SUBMISSION_URL` (optional)     | Sets custom URL for submitting logs | -             |

### .NET

Agentless log submission is supported for the following languages and frameworks:

-   `dd-trace-dotnet >= 2.50.0` and XUnit TestOutputHelper.

Use the following environment variables to enable and configure Agentless log submission:

| Name                                      | Description                                   | Default value |
| ----------------------------------------- | --------------------------------------------- | ------------- |
| `DD_CIVISIBILITY_LOGS_ENABLED` (required) | Enables/disables CI Visibility log submission | `false`       |

### Swift

Use the following environment variables to enable and configure log submission:

| Name                               | Description                            | Default value |
| ---------------------------------- | -------------------------------------- | ------------- |
| `DD_ENABLE_STDOUT_INSTRUMENTATION` | Enables/disables stdout log submission | `false`       |
| `DD_ENABLE_STDERR_INSTRUMENTATION` | Enables/disables stderr log submission | `false`       |

### Ruby

> Agentless logs submission with Test Optimization is supported for Rails applications.
> You need to [instrument your app with Datadog tracing][1] before enabling this feature.

Reviewer: ```suggestion
Agentless logs submission with Test Optimization is supported for Rails applications. Before enabling, ensure
that your application is [instrumented with Datadog tracing][1].
```

> Agentless log submission requires `datadog-ci` version `0.16` or newer.
> It supports the following logging libraries:

Reviewer: ```suggestion
To use agentless log submission, you need `datadog-ci` version `0.16` or later. The following logging libraries are supported:
```
